### PR TITLE
Check fixed for tax ID field in billing info

### LINKED ISSUE #4030: Billing Info form: FE check not working for Company field
**Describe the bug**

- FE check not working for company field
- `Tax ID or business ID` field not  marked as `*`

**Screenshots**
Screenshot from 2020-02-08 18-10-46

**Additional context**

Up For Grabs

### PR DESCRIPTION
**Pull request by aknrg77**
Fixes #4030

#### Short description of what this resolves:
Screenshot 2020-02-11 at 2 39 55 AM

#### Changes proposed in this pull request:

tax ID or business ID field marked as *
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by snitin315** [comment]
FE. Check for country field?

**Comment by aknrg77** [comment]
Sorry mistakenly wrote the wrong title ,now fixed it

**Comment by snitin315** [comment]
> Sorry mistakenly wrote the wrong title ,now fixed it

The title was correct I was trying to say that the issue is not completely fixed yet. Front End validation for country field is still missing.

**Comment by aknrg77** [comment]
validation for country field is there
Screenshot 2020-02-13 at 1 36 28 AM

**Comment by snitin315** [comment]
> validation for country field is there

Sorry , Company Field not Country 😅

File: app/components/forms/user-payment-info-form.js [hunk 1]
```diff
@@ -63,6 +63,15 @@ export default class extends Component.extend(FormMixin) {
             }
           ]
         },
+        taxID: {
```
Comment (snitin315) [inline app/components/forms/user-payment-info-form.js]: Unnecessary Change. Revert it.

Comment (aknrg77) [reply app/components/forms/user-payment-info-form.js]: Company field validation is also there
Screenshot 2020-02-14 at 1 11 01 AM
above code is for generating prompt for tax ID when it is left empty
here is the SS
Screenshot 2020-02-14 at 1 16 27 AM

Comment (snitin315) [reply app/components/forms/user-payment-info-form.js]: >above code is for generating prompt for tax ID when it is left empty

The prompt messege was already being generated, it was company field for which the messege was not being prompt . You can see the screenshot of corresponding issue.

Comment (snitin315) [reply app/components/forms/user-payment-info-form.js]: >Company field validation is also there

Can you share a working GIF?

**Comment by CosmicCoder96** [comment]
stale.
